stack/tasks/bundles/lib.js

```diff
@@ -51,6 +51,7 @@ components.bundleLibFiles({
       exclude: [
         'node_modules/**',
       ],
+      plugins: ['transform-runtime'],
```

Comment:
@LucaMele 
No I enabled the Babel's `transform-runtime` plugin.
https://babeljs.io/docs/plugins/transform-runtime/

Though we should check if we prefer `external-helpers` 🤔 
https://babeljs.io/docs/plugins/external-helpers/